smpplib/client.py

```diff
@@ -421,3 +422,75 @@ def query_message(self, **kwargs):
         qsm = smpp.make_pdu('query_sm', client=self, **kwargs)
         self.send_pdu(qsm)
         return qsm
+
+
+class ThreadSafeClient(Client):
+    should_stop = False
+
+    def __init__(
+        self,
+        *args,
+        select_timeout=1.0,
```

Comment:
I wonder how to fix it for python2.7
Copypasting Client.__init__ signature does not look correct
`kwargs.pop('select_timeout', 1.0)`?